docs/.vuepress/theme/components/SearchBox.vue

```diff
@@ -141,6 +152,12 @@ export default {
       const maxAPI = this.$site.themeConfig.searchMaxAPISuggestions || SEARCH_MAX_SUGGESTIONS;
       const maxGuides = this.$site.themeConfig.searchMaxGuidesSuggestions || SEARCH_MAX_SUGGESTIONS;
       const fuzzySearchDomains = this.$site.themeConfig.fuzzySearchDomains || [];
+      const APISearchDomainPriorityList = ([
```

Comment:
I would like to follow the rule that shortcuts with more than 2 letters are used same way as a word: `apiSearchDomainPriorityList`